build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
```

Comment:
Please move the code into the [minimal](https://github.com/spring-projects-experimental/spring-authorization-server/blob/master/samples/boot/minimal/spring-authorization-server-samples-boot-minimal.gradle) sample and remove this gradle build.